poi-ooxml/src/main/java/org/apache/poi/xwpf/model/XWPFHeaderFooterPolicy.java

```diff
@@ -52,227 +41,236 @@ Licensed to the Apache Software Foundation (ASF) under one or more
 import org.openxmlformats.schemas.wordprocessingml.x2006.main.STHdrFtr;
 import org.openxmlformats.schemas.wordprocessingml.x2006.main.STHdrFtr.Enum;
 
+import com.microsoft.schemas.office.office.CTLock;
+import com.microsoft.schemas.office.office.STConnectType;
+import com.microsoft.schemas.vml.CTFormulas;
+import com.microsoft.schemas.vml.CTGroup;
+import com.microsoft.schemas.vml.CTH;
+import com.microsoft.schemas.vml.CTHandles;
+import com.microsoft.schemas.vml.CTPath;
+import com.microsoft.schemas.vml.CTShape;
+import com.microsoft.schemas.vml.CTShapetype;
+import com.microsoft.schemas.vml.CTTextPath;
+import com.microsoft.schemas.vml.STExt;
+
 /**
- * A .docx file can have no headers/footers, the same header/footer
- * on each page, odd/even page footers, and optionally also
- * a different header/footer on the first page.
- * This class handles sorting out what there is, and giving you
+ * A .docx file can have no headers/footers, the same header/footer on each
+ * page, odd/even page footers, and optionally also a different header/footer on
+ * the first page. This class handles sorting out what there is, and giving you
  * the right headers and footers for the document.
  */
 public class XWPFHeaderFooterPolicy {
-    public static final Enum DEFAULT = STHdrFtr.DEFAULT;
-    public static final Enum EVEN = STHdrFtr.EVEN;
-    public static final Enum FIRST = STHdrFtr.FIRST;
-
-    private XWPFDocument doc;
-
-    private XWPFHeader firstPageHeader;
```

Comment:
please do not reformat the file - use the existing format so the changes can be evaluated - we use spaces for indents - not tabs.

Leave the imports in the same order

Don't reformat the existing javadocs

Reply:
Hello

I'll follow what you pointed. 

Is ok if I close this pull request and open another with the right formatted data? 

I'm new in coding, so I'm not sure what is the next step from my side, apart from fixing the data.

Reply:
You can create a new Pull Request if you prefer. Alternatively, you can add more commits to your existing branch and this Pull Request will be automatically updated.

Reply:
I did a new commit to the branch. 
Thanks for your support!